lib/v2/bilibili/userChannelCollections.js

```diff
@@ -0,0 +1,54 @@
+const got = require('@/utils/got');
+const cache = require('./cache');
+const utils = require('./utils');
+
+const notFoundData = {
+    title: '此 bilibili 频道不存在',
+};
+
+module.exports = async (ctx) => {
+    const uid = Number.parseInt(ctx.params.uid);
+    const sid = Number.parseInt(ctx.params.sid);
+    const disableEmbed = ctx.params.disableEmbed;
+
+    const link = `https://space.bilibili.com/${uid}/channel/collectiondetail?sid=${sid}`;
+    const userName = await cache.getUsernameFromUID(ctx, uid);
+    const host = `https://api.bilibili.com/x/polymer/space/seasons_archives_list?mid=${uid}&season_id=${sid}&sort_reverse=false&page_num=1&page_size=10`;
+
+    const response = await got({
+        method: 'get',
+        url: host,
+        headers: {
+            Referer: link,
+        },
+    });
```

Comment:
got request can be simplified to
```suggestion
    const response = await got(host, {
        headers: {
            Referer: link,
        },
    });
```